packages/package_info_plus/package_info_plus/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: package_info_plus
 description: Flutter plugin for querying information about the application package, such as CFBundleVersion on iOS or versionCode on Android.
-version: 1.4.0
+version: 1.4.1
```

Comment:
Could you sync the platform plugin dependency versions below?

```diff
diff --git a/packages/package_info_plus/package_info_plus/pubspec.yaml b/packages/package_info_plus/package_info_plus/pubspec.yaml
index 6d071b2..bcb616a 100644
--- a/packages/package_info_plus/package_info_plus/pubspec.yaml
+++ b/packages/package_info_plus/package_info_plus/pubspec.yaml
@@ -25,9 +25,9 @@ dependencies:
   flutter:
     sdk: flutter
   package_info_plus_platform_interface: ^1.0.2
-  package_info_plus_linux: ^1.0.3
+  package_info_plus_linux: ^1.0.4
   package_info_plus_macos: ^1.2.0
-  package_info_plus_windows: ^1.0.4
+  package_info_plus_windows: ^1.0.5
   package_info_plus_web: ^1.0.4
 
 dev_dependencies:
```

Reply:
😅 Done